OrbitGrpcProtos/capture.proto

```diff
@@ -226,9 +226,10 @@ message AddressInfo {
 }
 
 message ModulesUpdateEvent {
```

Comment:
Maybe rename to `ModuleUpdateEvent` now that it's only one?

Reply:
right. done.